docs/sphinx/source/emd_packages/grasp_planner/grasp_planner_input.rst

```diff
@@ -85,7 +85,7 @@ Greater hardware requriements: EPD Workflow
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Due to the higher hardware limitations for deploying of deep learning models, hardware requirements will be higher for the EPD workflow.
```

Comment:
Unrelated to the change in this PR, I haven't found where the info this paragraph suggests to refer to can be found. If anyone can share the location (e.g. URL on readthedocs.io or folder name on git repo) then I can include the fix in this PR. The URL this paragraph is linking is a top page of EPD and not super helpful.